latex/Inhalt/design.tex

```diff
@@ -1 +1,52 @@
-\chapter{Design}
+\chapter{Design}\label{chap:design}
+
+\section{Emulator}
+
+\subsection{Zentrale Struktur}
+
+Den Kern des Emulators bildet eine Struktur, welche zuständig für die Ausführung der Maschinencode-Programme ist. Diese Struktur gruppiert alle notwendigen Komponenten eines Intel 8080 Systems. Der Aufbau der Struktur ist in \cref{core} illustriert.
```

Comment:
The listing does not include the number given in the text, is this expected behaviour / design?